### DISCUSSION TITLE
Welcome to UsualScrap Discussions!

### DISCUSSION THREAD

**Discussion by EtherealEmil** [EtherealEmil/UsualScrap#11] [Announcements]
This is the discussions tab! Feel free to create a thread for anything that isn't a bug or issue here and I'll respond to it ASAP.

**Comment by GhostsModels** [comment]
For candy you could talk to Xu Xiaolan and make it so that candy doesn't take up the item spawn pool like with coins in the coderebirth mod

**Reply by GhostsModels** [reply]
oh, and you should also probably make a page on the modding discord, you are more likely to get feedback there

**Reply by EtherealEmil** [reply]
I don't know who that is but that'd be cool. I'll check out the discord.

**Comment by GhostsModels** [comment]
Your changelogs for 1.4.9 and 1.5.7 are the same as the ones previous to them

**Reply by GhostsModels** [reply]
you should also be able to change the mod name on thunderstore as long as you don't change the GUID without causing problems (I believe)

**Reply by EtherealEmil** [reply]
The change logs are the same probably because I just forgot one thing and updated again within a very short time but didn't want to *cover* the previous changes.

**Reply by EtherealEmil** [reply]
For now the name will stick around, the mod's name isn't great but it'll do. I may add a space to it though.
